### PR TITLE
feat : trash 

### PR DESCRIPTION
**Pull request by arpit01923**
Functionalities :
- add to trash
- restore from trash
- delete from trash

Files to check :
- src/components/notecontainer/notecontainer.jsx
- src/pages/trash/trash.jsx
- src/reducer/notearray.jsx

### CODE REVIEW

File: src/components/notecontainer/notecontainer.jsx [hunk 1]
```diff
@@ -43,19 +43,65 @@ export const NoteContainer = ({ title, text, color, date, time, id }) => {
   const removeArchiveNote = async () => {
     try {
       const response = await axios.post(
-        `/api/archives/restore/${id}`,
+        `/api/archives/restore/${_id}`,
         {},
```
Comment (anupkgurung) [inline src/components/notecontainer/notecontainer.jsx:47]: why are you giving an empty object?

**Review by anupkgurung — APPROVED** [review]
code is clean and clear.
just not sure what is the use of empty object when using post request.
one suggestion in login & signup page you can lighten up a the border or give some shadow instead of solid border.

**Comment by govardhanreddyc79** [comment]
Clean code,
Good to merge.

File: src/components/notecontainer/notecontainer.jsx (hunk 1 above)
Comment (arpit01923) [reply src/components/notecontainer/notecontainer.jsx:47]: it's a way in mockbee to restore data from the API. We have to give an empty object in request body

**Pull request merged**